.gitignore

```diff
@@ -1,34 +1,4 @@
 *.DS_Store
-
-# Prerequisites
-*.d
-
-# Compiled Object files
-*.slo
-*.lo
-*.o
-*.obj
-
-# Precompiled Headers
-*.gch
-*.pch
-
-# Compiled Dynamic libraries
-*.so
-*.dylib
-*.dll
-
-# Fortran module files
-*.mod
-*.smod
-
-# Compiled Static libraries
-*.lai
-*.la
-*.a
-*.lib
-
-# Executables
-*.exe
-*.out
-*.app
+build
```

Comment:
FWIW you don't need to explicitly ignore the build directory.

https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs